polling_stations/apps/file_uploads/api.py

```diff
@@ -59,9 +59,9 @@ def save(validated_data):
             if (
                 file_set.first().ems == "Democracy Counts" and file_set.count() == 2
             ) or file_set.first().ems != "Democracy Counts":
-                user = self.context["request"].user
                 upload.make_pull_request()
-                upload.send_confirmation_email(user=user)
+                user = upload.gss.users.first()
```

Comment:
I don't feel like this is the best way forward and would welcome suggestions for a more reliable alternative. I had hoped to pull in the request object and get the user that way, but struggled to do this.